docs: update optimization and source map section

alan-agius4 opened the pull request:
With this change we split the optimization and source map configuration section section into two, improve it overall by adding the options a table and add the new options (`inlineCritical` and `minify`) that have been introduced in CLI version 11.1

mary-poppins commented:
You can preview 809fbd2 at https://pr40037-809fbd2.ngbuilds.io/.

petebacondarwin commented:
Here is a link to the preview of the changed guide: https://pr40037-809fbd2.ngbuilds.io/guide/workspace-config

mary-poppins commented:
You can preview 71bbb5b at https://pr40037-71bbb5b.ngbuilds.io/.

alan-agius4 committed:
docs: update optimization and source map section

With this change we split the optimization and source map
configuration section section into two, improve it overall by adding
theoptions a table and add the new options
(`inlineCritical` and `minify`) that have been introduced in
CLI version 11.1

mary-poppins commented:
You can preview 5bde243 at https://pr40037-5bde243.ngbuilds.io/.

alan-agius4 commented:
Link to guide  https://pr40037-5bde243.ngbuilds.io/guide/workspace-config

alan-agius4 committed:
fixup! docs: update optimization and source map section

mary-poppins commented:
You can preview 74aa2e4 at https://pr40037-74aa2e4.ngbuilds.io/.

aikithoughts reviewed (approved):
Reviewed-for: global-docs-approvers

kyliau commented on aio/content/guide/workspace-config.md:
```diff
@@ -361,34 +361,156 @@ Files in that folder, such as `src/style-paths/_variables.scss`, can be imported
+</table>
+
+
+You can supply a value such as the following to output source maps to one or the other:
```
```suggestion
The example below shows how to toggle one or more values to configure the source map outputs:
```

alan-agius4 commented on aio/content/guide/workspace-config.md:
```diff
@@ -361,34 +361,156 @@ Files in that folder, such as `src/style-paths/_variables.scss`, can be imported
+
+There are several options that can be used to fine-grained the optimization of an application.
+
+<table class="is-full-width is-fixed-layout">
```
I don't know, I just followed the style of other tables in AIO.

alan-agius4 commented on aio/content/guide/workspace-config.md:
```diff
@@ -361,34 +361,156 @@ Files in that folder, such as `src/style-paths/_variables.scss`, can be imported
-* The `optimization` option applies to scripts, styles and fonts. You can supply a value such as the following to apply optimization to one or the other:
+### Optimization configuration
+
+The `optimization` browser builder option can be either a Boolean or an Object for more fine-grained configuration. This option enables various optimizations of the build output.
```
Much better!

mary-poppins commented:
You can preview 962d36e at https://pr40037-962d36e.ngbuilds.io/.

mary-poppins commented:
You can preview 9e7bc9b at https://pr40037-9e7bc9b.ngbuilds.io/.

alan-agius4 commented:
Link to guide https://pr40037-e1c5fb9.ngbuilds.io/guide/workspace-config

mary-poppins commented:
You can preview e1c5fb9 at https://pr40037-e1c5fb9.ngbuilds.io/.

alan-agius4 committed:
fixup! docs: update optimization and source map section

mary-poppins commented:
You can preview 1df3598 at https://pr40037-1df3598.ngbuilds.io/.
